src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java

```diff
@@ -239,8 +239,9 @@ public enum CaseEvent {
     TRIGGER_UPDATE_GA_LOCATION(CAMUNDA),
     NOTIFY_RESPONDENT_SOLICITOR1_FOR_BUNDLE_CREATED(CAMUNDA),
     NOTIFY_RESPONDENT_SOLICITOR2_FOR_BUNDLE_CREATED(CAMUNDA),
-    NOTIFY_APPLICANT_SOLICITOR1_FOR_BUNDLE_CREATED(CAMUNDA);
-
+    NOTIFY_APPLICANT_SOLICITOR1_FOR_BUNDLE_CREATED(CAMUNDA),
+    NOTIFY_LIP_CLAIMANT_HEARING(CAMUNDA),
```

Comment:
Are these being used? I thought we were just using the pre-existing case events.

Reply:
I forgot to remove it, thanks!